lib/components/split-pane.js

```diff
@@ -18,6 +20,28 @@ export default class SplitPane extends PureComponent {
     }
   }
 
+  setupPanes(ev) {
+    this.panes = Array.from(ev.target.parentNode.childNodes);
+    this.paneIndex = this.panes.indexOf(ev.target);
```

Comment:
(This is not your code originally but..)
paneIndex is valide/set only after a dragStart.
Maybe better to have a name like : `this.draggingPaneIndex`

But if we use variable when rendering, we don't need it anymore